docker/install/ubuntu_install_arduino.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+set -u
+set -o pipefail
+
+export DEBIAN_FRONTEND=noninteractive
+apt-get update --fix-missing
+apt-get install -y ca-certificates
+
+# Install arduino-cli latest version
+wget -O - https://raw.githubusercontent.com/arduino/arduino-cli/master/install.sh | sh -s
+
+# Board manager URLs, separated by semicolons
+BOARD_MANAGER_URLS=https://github.com/sonydevworld/spresense-arduino-compatible/releases/download/generic/package_spresense_index.json
```

Comment:
should we list these in the CLI config? https://arduino.github.io/arduino-cli/latest/getting-started/#adding-3rd-party-cores

then we don't have to worry about forgetting this option later on. (or do we want to not worry about this? we don't want the CI to depend on some network resource that could become unavailable)

Reply:
Tweaked a little to make it clearer what depends on what. We only need to specify the `--additional-urls` flag when we run `arduino-cli core install` right below - after that, further commands don't need the flag or a network connection.